apps/docs/src/components/HomeLayout/index.tsx

```diff
@@ -20,11 +22,23 @@ export const HomeLayout: FC<Props> = ({ docs, params, docsTree }) => {
 	}
 
 	return (
-		<Fragment>
-			<TopNavigation docs={docs} docsTree={docsTree} />
-			<Content docsTree={docsTree} docs={docs} params={params} />
-		</Fragment>
+		<View style={styles.container}>
+			<View style={sharedStyles.container}>
+				<TopNavigation docs={docs} docsTree={docsTree} />
+			</View>
+			<ScrollView showsVerticalScrollIndicator={false}>
+				<View style={sharedStyles.container}>
+					<Content docsTree={docsTree} docs={docs} params={params} />
+				</View>
+			</ScrollView>
```

Comment:
use this to avoid create extra `<View/>` Node: 
```typescript
<ScrollView contentContainerStyle={sharedStyles.container}>
  <Content docsTree={docsTree} docs={docs} params={params} />
</ScrollView>
```